frontend/src/app/conference/conference-detail/conference-detail.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import {Conference} from '../shared/conference.model';
+import {PcMember} from "../../user/pc-member/shared/pcMember.model";
+import {MEMBERS} from "../../user/pc-member/shared/mock-members";
+
+@Component({
+  selector: 'app-conference-detail',
+  templateUrl: './conference-detail.component.html',
+  styleUrls: ['./conference-detail.component.scss']
+})
+export class ConferenceDetailComponent implements OnInit {
+
+  conference: Conference = {
```

Comment:
please create a conference service and return an observable of this mock. this way we will need to only change the service when implementing the backend stuff, without changing anything in the component itself